source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -3,7 +3,7 @@ title: "Configuring Hiera"
 ---
 
 [hierarchy]: ./hiera_intro.html#Hiera-hierarchies
-[layers]: ./hiera_intro.html#Hiera’s-three-config-layers
+[layers]: ./hiera_intro.html#Hiera's-three-config-layers
```

Comment:
You might need to test to confirm that this anchor works with the apostrophe in it.